trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk

```diff
@@ -1,100 +1,137 @@
-                "data-cy":"add-credit-rating-link"
-              }
-            }) }}
-          {% endif %}
-        </td>
-
-      </tr>
-
-      <tr class="govuk-table__row">
-
-        <td scope="row" class="govuk-table__header govuk-body-s govuk-table__cell width-half" data-cy="exporter-table-loss-given-default-heading">
-          Loss given default
-        </td>
-
-        <td class="govuk-table__cell" data-cy="exporter-table-loss-given-default-value">
-          {% if lossGivenDefault %}
-            {{ lossGivenDefault }}%
-          {% else %}
-            -
-          {% endif %}
-        </td>
-
-        <td class="govuk-table__cell govuk-table__cell--border-top text-align-right">
-          {% if userCanEditGeneral %}
-            <a class="govuk-link" href="/case/{{ caseId }}/underwriting/pricing-and-risk/loss-given-default" data-cy="exporter-table-change-loss-given-default-link">Change</a>
-          {% endif %}
-        </td>
-
-      </tr>
-
-      <tr class="govuk-table__row">
-
-        <td scope="row" class="govuk-table__header govuk-body-s govuk-table__cell width-half" data-cy="exporter-table-probability-of-default-heading">
-          Probability of default
-        </td>
-
-        <td class="govuk-table__cell" data-cy="exporter-table-probability-of-default-value">
-
-          {% if probabilityOfDefault %}
-            Less than {{ probabilityOfDefault }}%
-
-            {% else %}
-            -
-          {% endif %}
-        </td>
-
-        <td class="govuk-table__cell govuk-table__cell--border-top text-align-right">
-          {% if userCanEditGeneral %}
-            <a class="govuk-link" href="/case/{{ caseId }}/underwriting/pricing-and-risk/probability-of-default" data-cy="exporter-table-change-probability-of-default-link">Change</a>
-          {% endif %}
-        </td>
-      </tr>
-
-    </tbody>
-  </table>
+  
+  {% set exporterCreditRatingHeadingHtml %}
+    <span data-cy="exporter-table-credit-rating-heading">Credit rating</span>
+  {% endset %}
+
+  {% if exporterCreditRating %}
+    {% set exporterCreditRatingValueHtml %}
+      <span data-cy="exporter-table-credit-rating-value">{{ exporterCreditRating }}</span>
+    {% endset %}
+    {% set editExporterCreditRatingActionText = 'Change' %}
+  {% else %}
+    {% set exporterCreditRatingValueHtml %}
+      {{ govukTag({
+        text: 'Not added',
+        classes: 'govuk-tag--yellow',
+        attributes: {
+          'data-cy': 'exporter-table-credit-rating-not-added'
+        }
+      }) }}
+    {% endset %}
+    {% set editExporterCreditRatingActionText = 'Add' %}
+  {% endif %}
+
+  {% if userCanEditGeneral %}
+    {% set creditRatingRow = {
+      key: { html: exporterCreditRatingHeadingHtml },
+      value: { html: exporterCreditRatingValueHtml },
+      actions: {
+        items: [
+          {
+            href: "/case/" + caseId + "/underwriting/pricing-and-risk/edit",
+            text: editExporterCreditRatingActionText,
```

Comment:
`Add/Change` context is clear for people with sight, but for screen readers we should add `visuallyHiddenText` or `<span class="govuk-visually-hidden">`.

Requirement example - https://ukef-dtfs.atlassian.net/browse/DTFS2-6467